data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 set -e
 
+## FIXME: CVO should use `/etc/kubernetes/bootstrap-secrets/kubeconfig` instead
```

Comment:
CVO should not be using `/etc/kubernetes/bootstrap-secrets/kubeconfig` that's for the bootstrap control-plane